src/ui/flowgraph/datasink.h

```diff
@@ -12,23 +12,33 @@ class DataSink final : public Block {
 public:
```

Comment:
Couldn't this re-use @frankosterfeld DataSink implementation: https://github.com/fair-acc/graph-prototype/blob/main/blocks/basic/include/gnuradio-4.0/basic/DataSink.hpp